### ISSUE TITLE
Provide an example illustrating two-level indices idea

### ISSUE THREAD

**Issue by arsenius7** [adevress/syn2_spec#2]
Also, it would be useful to state in the spec the rationale behind it (getting the connectivity of a neuron in two reads).
And please clarify what does "O(log(S)) maximum time complexity" correspond to.

**Comment by adevress** [comment]
Yep, I will try to add a more informative message on this point.

It is not trivial to explain briefly the motivation of this type of indexing, but we should at least make it a bit easier to understand.
